### PR TITLE
Add Python 3.9 to build

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Review by KevinHock — APPROVED** [review]
Thank you @gliptak!!

**Comment by gliptak** [comment]
@KevinHock we are waiting for somebody to run the deploy build

https://github.com/travis-ci/cpython-builder/pull/33

https://travis-ci.community/t/python-3-9-0-build/10091/10

**Comment by KevinHock** [comment]
I see @cclauss where ever I go lol

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ matrix:
         - env: TOXENV=py38
           python: 3.8
           dist: xenial  # Required for Python >= 3.7 (travis-ci/travis-ci#9069)
+        - env: TOXENV=py39
+          python: 3.9
```
Comment (cclauss) [inline .travis.yml:15]:
```suggestion
          python: 3.9-dev
```

File: .travis.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ matrix:
...
+          dist: focal  # Required for Python >= 3.7 (travis-ci/travis-ci#9069)
```
Comment (cclauss) [inline .travis.yml:16]:
```suggestion
          dist: focal
```
That issue was closed when Xenial became the default distro

File: .travis.yml [hunk 3]
```diff
@@ -11,6 +11,9 @@ matrix:
         - env: TOXENV=py38
           python: 3.8
           dist: xenial  # Required for Python >= 3.7 (travis-ci/travis-ci#9069)
```
Comment (cclauss) [inline .travis.yml:13]:
```suggestion
          dist: xenial
```

**Comment by cclauss** [comment]
Replace `matrix` with `jobs`, etc...
Screenshot 2020-10-26 at 15 07 27

**Comment by gliptak** [comment]
@cclauss updated
image

File: .travis.yml [hunk 4]
```diff
@@ -10,7 +12,10 @@ matrix:
...
```
Comment (cclauss) [inline .travis.yml:12]:
```suggestion
```
Already done on line 3.

**Comment by cclauss** [comment]
Someone with Travis admin rights may need to go to https://travis-ci.org/github/Yelp/detect-secrets and under `More options` select `Trigger build` on master to get the tests going again.  If that does not work, it might be necessary to [migrate from travis-ci.org --> travis-ci.com](https://docs.travis-ci.com/user/migrate/open-source-repository-migration)

**Review by cclauss — APPROVED** [review]
LGTM

**Comment by gliptak** [comment]
https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/20.04/x86_64/python-3.9.tar.bz2

**Comment by cclauss** [comment]
Nice that Travis no’s supports Python 3.9!!

Given the slow execution times, it might still be necessary to [migrate from travis-ci.org --> travis-ci.com](https://docs.travis-ci.com/user/migrate/open-source-repository-migration)

**Pull request merged**